taskt/Core/Automation/Commands/DataTable Commands/GetDataRowCountCommand.cs

```diff
@@ -1,90 +1,90 @@
 ﻿using System;
-using System.Linq;
-using System.Xml.Serialization;
+using System.Collections.Generic;
 using System.Data;
+using System.Linq;
 using System.Windows.Forms;
-using System.Collections.Generic;
-using taskt.UI.Forms;
-using taskt.UI.CustomControls;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Script;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("DataTable Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to get the datarow count of a DataTable")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to get the datarow count of a DataTable.")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("DataTable Commands")]
+    [Description("This command gets the DataRow Count of a DataTable.")]
+
     public class GetDataRowCountCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the DataTable Name")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter a existing DataTable.")]
-        [Attributes.PropertyAttributes.SampleUsage("**myData**")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        public string v_DataTableName { get; set; }
+        [PropertyDescription("DataTable")]
+        [InputSpecification("Enter a existing DataTable.")]
```

Comment:
I think It should be "Enter **an** existing DataTable".
Please confirm.

Reply:
done